doc/intro.rst

```diff
@@ -177,6 +177,18 @@ strings. The following fields are recognized:
   makes no judgement about its value.
 * ``job_url``: URL pointing to the original test run.
 * ``resubmit_url``: URL that can be used to resubmit the test run.
+* ``suite_versions``: a dictionary with version number strings for suite names
```

Comment:
How does this work if there are multiple versions of a given test suite across projects? (or what is the object model - does it belong to each project? i thought it was independent of projects)

Reply:
the version is recorded at the level of the test run, so you can even have different versions of some test suite executed against the same build.